fandogh_cli/service_commands.py

```diff
@@ -157,16 +157,16 @@ def service_details(service_name):
 
 @click.command('apply', cls=FandoghCommand)
 @click.option('-f', '--file', 'file', prompt='File address')
-def service_apply(file):
+@click.option('-p', '--parameter', 'parameters', help='Manifest parameters', multiple=True)
+def service_apply(file, parameters):
     """Deploys a service defined as a manifest"""
-    try:
-        with open(file, mode='r') as manifest:
-            manifest_content = manifest.read()
-            click.echo(manifest_content)
-    except FileNotFoundError as e:
-        click.echo(format_text(e.strerror, TextStyle.FAIL), err=True)
+    manifest_content = read_manifest(file, parameters)
+    if manifest_content is None:
+        return
+    click.echo(manifest_content)
+    if not click.confirm("Please review the manifest, Are you sure you want to deploy this manifest?"):
```

Comment:
what is it for? was it in the scope of the task?

Reply:
We had to add this extra confirmation since we're rendering user's manifest, it's better to ask user to review the rendered manifest.